test-data/local/poly_example/assert_runpath_file.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+import sys
+import os
+
+
+def run():
```

Comment:
Should we do `run(runpath_file, curdir)` and then `run(sys.argv[1], sys.argv[2])`?